cmd/rekor-cli/app/pflag_groups.go

```diff
@@ -73,6 +73,11 @@ func addArtifactPFlags(cmd *cobra.Command) error {
 			"path or URL to public key file",
 			false,
 		},
+		"multi-public-key": {
```

Comment:
What are your thoughts on just changing `public-key` to take multiple values instead of creating an entirely new cmd line argument?

Reply:
Yes, I agree with this path. I actually started to go down this route but wanted the maintainers feedback before I changed an existing flag. Changing `public-key` would be the better path, I did not want to break something unintentionally because of it.

Reply:
Changed.